### PR TITLE
Allow Last Responder thread pointer to be NULL

### PR DESCRIPTION
**Pull request by dmitripivkine**
Last Responder thread can be NULL in the case of Safe Point Exclusive. Check pointer for NULL and leave thread name empty.

### CODE REVIEW

**Comment by dmitripivkine** [comment]
@amicic @babsingh This PR is ready for review.

**Comment by dmitripivkine** [comment]
related https://github.com/eclipse-openj9/openj9/issues/18260

**Review by amicic — APPROVED** [review]

File: gc/verbose/VerboseHandlerOutput.cpp [hunk 1]
```diff
@@ -609,7 +613,8 @@ MM_VerboseHandlerOutput::handleExclusiveStart(J9HookInterface** hook, uintptr_t
 	}	
 	writer->formatAndOutput(env, 0, "<exclusive-start %s intervalms=\"%llu.%03.3llu\">", tagTemplate, deltaTime / 1000, deltaTime % 1000);
 	writer->formatAndOutput(env, 1, "<response-info timems=\"%llu.%03.3llu\" idlems=\"%llu.%03.3llu\" threads=\"%zu\" lastid=\"%p\" lastname=\"%s\" />",
```
Comment (babsingh) [inline gc/verbose/VerboseHandlerOutput.cpp:615]: https://github.com/eclipse/omr/blob/master/doc/CodingStandard.md#function-parameter-list-does-not-fit-on-one-line

> - Start the parameter list on a new line. Insert a newline immediately after the opening bracket.
> - Indent the parameter list by at least 1 tab. Where reasonable, the parameter list should be aligned with the opening bracket.
> - Use the same indent for each line of parameters.

**Comment by babsingh** [comment]
jenkins build all

**Review by babsingh — APPROVED** [review]

**Pull request merged**